README.md

```diff
@@ -2,55 +2,158 @@
 
 ![Dolphin Guy](http://i.giphy.com/RdkwfjGlsdnRm.gif)
 
-## Learning Objectives - The student should be able to..
+## Objectives
 
-* Explain that the `Bool` is a data type where the value can only be **true** or **false**.
-* Create variables using the Boolean literal values, `true` or `false` like so:
+1. Describe the `Bool` data type
+1. Create Boolean variables with literal values
+1. Use comparison operators on Boolean variables
+1. Use logic operators on Boolean variables
+
+## Understanding Booleans
+
+You are already familiar with creating `Int` variables and constants, which can store single numeric integer values, like
```

Comment:
Also helpful to quickly mention Strings too so that they are aware too of the other data type that was covered, and also explicitly using the word "data type."